public/css/passwordindicator.css

```diff
@@ -0,0 +1,83 @@
+
+.password-wrapper {
+    height: 45px;
+    width: 100%;
+    display: flex;
+    position: relative;
+}
+form .password-wrapper .password{
```

Comment:
Nice work, it will be better if you add a single line after the ending of each css block